pkg/client/client.go

```diff
@@ -25,6 +25,7 @@ type Deployment struct {
 	ID                      string   `json:"id"`
 	OSName                  string   `json:"osname"`
 	Serial                  int32    `json:"serial"`
+	BaseChecksum            *string  `json:"base-checksum"`
```

Comment:
Any reason we have `*` here when others don't?

Reply:
This field is not always set.  Go doesn't have `Option<T>` so this is the usual way to represent optional values.